src/sidebar/store/modules/test/drafts-test.js

```diff
@@ -0,0 +1,200 @@
+'use strict';
+
+const immutable = require('seamless-immutable');
+
+const drafts = require('../drafts');
+const { Draft } = require('../drafts');
+const createStore = require('../../create-store');
+
+const fixtures = {
+  draftWithText: immutable({
+    isPrivate: false,
+    text: 'some text',
+    tags: [],
+  }),
+  draftWithTags: immutable({
+    isPrivate: false,
+    text: '',
+    tags: ['atag'],
+  }),
+  emptyDraft: immutable({
+    isPrivate: false,
+    text: '',
+    tags: [],
+  }),
+
+  annotation: immutable({
+    id: 'my_annotation',
+    $tag: 'my_annotation_tag',
+  }),
+};
+
+describe('Drafts Store', () => {
+  let store;
+
+  beforeEach(() => {
+    store = createStore([drafts]);
+  });
+
+  describe('Draft', () => {
+    it('creates a non empty draft object', () => {
```

Comment:
The structure of the `describe` blocks isn't quite right here, as the `describe` blocks for the methods are outside the one for the `Draft` class. For a class it should typically look something like this:

```js
describe("SomeClassName", () => {
  describe('constructor', () => {

  });
  
  // Test for a specific method
  describe("#someMethod", () => {
  ...
  });

  // Test not about a specific method
  it("should ...", () => {
    
  });
});

describe('someOtherFunction', () => {
  ...
});
```